modules/cluster-logging-deploy-cli.adoc

```diff
@@ -139,14 +139,14 @@ metadata:
   name: "elasticsearch-operator"
   namespace: "openshift-operators-redhat" <1>
 spec:
-  channel: "5.0" <2>
+  channel: "stable-5.1" <2>
   installPlanApproval: "Automatic"
   source: "redhat-operators" <3>
   sourceNamespace: "openshift-marketplace"
   name: "elasticsearch-operator"
 ----
 <1> You must specify the `openshift-operators-redhat` Namespace.
-<2> Specify `5.0` or `stable` as the channel. `stable` is the latest 5.x released code.
```

Comment:
I don't know if it matters or we need to document, but there will be a "5.0" channel.  To meet suggested guidelines we introduced the stable prefix but intentionally did not apply it to  5.0  for backwards compatibility

Reply:
Updated. Thanks.